components/global-header/GlobalHeader.tsx

```diff
@@ -28,13 +28,8 @@ export const GlobalHeader = ({}: IProps) => {
   return (
     <div className="flex justify-between items-center">
       <Link href={'/'} title="mumble">
-        <div className="w-10 h-10 md:hidden">
-          <Icon
-            type={EIConTypes.MUMBLE}
-            fitParent={true}
-            // extend to icon colo white or inherit color
-            color={EIconColors.VIOLET}
-          />
+        <div className="w-10 h-10 md:hidden text-white">
```

Comment:
tip: you can also set `size-10` instead of `w-10 h-10`  (https://tailwindcss.com/docs/size)